README.md

```diff
@@ -39,7 +39,7 @@ DATA=$(printf '{"database":"projects/[PROJECT_ID]/instances/[INSTANCE_ID]/databa
 
 ## Deploy scheduled jobs to Cloud Scheduler
 
-Note: To use Cloud Scheduler, we must [create an App Engine app](https://cloud.google.com/scheduler/docs#supported_regions).
+Note: To use Cloud Scheduler, we must [create an App Engine app](https://cloud.google.com/scheduler/docs#supported_regions). Cloud Scheduler must be deployed in the same region with App Angine. To specify a region, add a `location` flag to the command below (default is set to `us-central1`).
```

Comment:
1. "App Angine" should be "App Engine". 
2. "the command below" is not very clear which command is referred here. Can we change to be more specific?

Reply:
Done.